Mathlib/Algebra/Order/Ring/Lemmas.lean

```diff
@@ -5,6 +5,7 @@ Authors: Damiano Testa, Yuyang Zhao
 -/
 import Mathlib.Algebra.CovariantAndContravariant
 import Mathlib.Algebra.GroupWithZero.Defs
+import Mathlib.Tactic.GCongr.Core
```

Comment:
Is there a reason not to simply import `Mathlib.Tactic.GCongr` here? That is, do we really not want to import `Mathlib.Tactic.Positivity`(`.Core`) here?

Reply:
I'm good with importing positivity as early as possible. Should I ask on Zulip for other opinions?

Reply:
I see that @semorrison recently removed some imports of `positivity`. Scott, what do you think about this?

Reply:
@j-loreaux, it would cause an import cycle!

Reply:
So, should I merge `master`, wait for CI and merge?

Reply:
Yes, go ahead and do that.

Reply:
> Yes, go ahead and do that.

I'm waiting for an official `bors d+` from another maintainer.

Reply:
There's #10140 in the mix now. Would you like to adapt your PR to that? (cc @YaelDillies)